universal-login-react/src/App.tsx

```diff
@@ -142,14 +142,19 @@ export const App = () => {
                 const topUpProps: TopUpProps = {
                   contractAddress: Wallet.createRandom().address,
                   onGasParametersChanged: console.log,
-                  sdk
+                  sdk,
+                  isDeployment: false
                 };
                 return (
                   <>
                     <ReactModalContext.Provider value={modalService}>
                       <button id="show-topup-button" onClick={() => modalService.showModal('topUpAccount', topUpProps)}>Show Topup</button>
                       <Modals />
                     </ReactModalContext.Provider>
+                    <ReactModalContext.Provider value={modalService}>
```

Comment:
What's the reason for adding a second modal provider and a second `Modals` component?

Reply:
I see your point, updated.